widget/ui/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,76 @@
+import type { PropTypes } from './Tabs.types';
+
+import React, { useEffect, useRef, useState } from 'react';
+
+import { Divider } from '../Divider';
+import { Tooltip } from '../Tooltip';
+
+import { BackdropTab, Tab, Tabs } from './Tabs.styles';
+
+export function TabsComponent(props: PropTypes) {
+  const {
+    items: tabItems,
+    onChange,
+    container = document.body,
+    value,
+    type,
+    borderRadius,
+  } = props;
+  const [tabWidth, setTabWidth] = useState(0);
+  const tabRef: React.Ref<HTMLButtonElement> = useRef(null);
+  const currentTabIndex = tabItems.findIndex((item) => item.id === value);
+
+  useEffect(() => {
+    const updateTabWidth = () => {
+      if (tabRef.current) {
+        const tabRect = tabRef.current.getBoundingClientRect();
+        setTabWidth(tabRect.width);
+      }
+    };
+    updateTabWidth();
+    window.addEventListener('resize', updateTabWidth);
+    return () => {
+      window.removeEventListener('resize', updateTabWidth);
+    };
+  }, []);
+
+  return (
+    <Tabs type={type} borderRadius={borderRadius}>
```

Comment:
I wanted to use it in the dapp and couldn't change its styles. Please define a style props for it.
(alternative way: You don't have to define a new prop. you can just pass 
```
const {
    items: tabItems,
    onChange,
    container = document.body,
    value,
    type,
    borderRadius,
...rest
  } = props;
```
and pass `rest` to <Tabs to allow users to use `styled(Tabs , {...` for overriding styles.

Reply:
I added className to this, like typography